README_dev.md

```diff
@@ -130,12 +131,14 @@ npm run test:local -- testSuiteName
       },
       "database": { "driver": "lokijs", "folder": "./" },
       "logging": { "categories": { "default": { "appenders": ["console"] } } },
-      "infrastructure": { "httpServer": { "apiKey": "xxx", port: 8080 } },
+      "infrastructure": { "httpServer": { "apiKey": "xxx", "port": 8080 } },
       "modules": { "coreHttpApi": { "docs": { "enabled": true } } }
     }
+
```

Comment:
If allowed by automatic formatting, this empty line can be removed, as there is also no empty line before this block of code starts (line 124/125).